Add missing Expression Language values (#2568)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

| Active Directory UPN                                               | `hasDirectoryUser() ? findDirectoryUser().managerUpn : null`                                                                                             | bob@okta.com            | Check if user has an Active Directory assignment, and if so, return their Active Directory manager UPN.                                                                                                                                                                                                                                                                                                |

## Expressions for custom claims

Okta provides a few expressions that you can only use with custom claims.

> * See [Create claims](/docs/guides/customize-authz-server/create-claims) for general information about claim creation.

Reviewer: ```suggestion
* See [Create claims](/docs/guides/customize-authz-server/create-claims)
```

> * See [Include app-specific information in a custom claim](/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute) for details about using app-specific values.

Reviewer: ```suggestion
* See [Include app-specific information in a custom claim](/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute)
```

| Active Directory UPN                                               | `hasDirectoryUser() ? findDirectoryUser().managerUpn : null`                                                                                             | bob@okta.com            | Check if user has an Active Directory assignment, and if so, return their Active Directory manager UPN.                                                                                                                                                                                                                                                                                                |

> ## Expressions for custom claims

Reviewer 2: Expressions for custom OAuth2/OIDC claims

> Okta provides a few expressions that you can only use with custom claims.

Reviewer 2: Oauth2/OIDC custom claims

* See [Create claims](/docs/guides/customize-authz-server/create-claims).
* See [Include app-specific information in a custom claim](/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute).

| Syntax           | Definitions                                                             | Examples     |
| --------         | ----------                                                              | ------------ |
| `app.$attribute` | `app` refers to the name of the OIDC app.<br>`$attribute` refers to the attribute variable name. | app.id<br>app.clientId<br>app.profile |
| `access.scope` | `access` refers to the access token requesting the Scopes.<br>`scope` refers to the array of granted Scopes. | access.scope |

### Samples

> #### Sample using an app rule

Reviewer 2: `Sample using app attributes` instead of `Sample using an app rule`. `rule` has special meaning in Okta, for example - group rules. But it does not apply here.

> > **Note:** The application reference is usually the `name` of the application, as distinct from the `label` (display name). See [Application properties](/docs/reference/api/apps/#application-properties). If your organization configures multiple instances of the same application, the names of the later instances are differentiated by a randomly assigned suffix, for example: `zendesk_9ao1g13`.  You can find the name of any specific app instance in the Profile Editor, where it appears in lighter text beneath the label of the app.

Reviewer 2: This note suggests you can use explicit references to app, like zendesk_9ao1g13.clientId.

However, earlier in the document there is a note which says `Note: Explicit references to apps aren't supported for custom OAuth/OIDC claims.`

Author: At your suggestion, I'll remove the note. Thanks!

> To catch an app Profile label, use the following expression:<br>

Reviewer 2: To `include` and app Profile label, instead of `catch`

`app.profile.label`

See [Include app-specific information in a custom claim](/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute).

> #### Sample using an access rule

Reviewer 2: **Sample using `access.scope`** instead of **Sample using an app rule**.

> To catch a granted Scope array and convert it to a space-delimited string, use the following expression:<br>

Reviewer 2: To `include` a granted scope array, instead of `catch`

> To include a granted Scope array and convert it to a space-delimited string, use the following expression:<br>

Reviewer 2: You can name a claim `scope` in API Access Management custom authorization servers. You can then use the EL expression `access.scope` in custom claims to return an array of granted scope strings.

To include a granted Scope array and convert it to a space-delimited string, use the following expression:<br>
`String.replace(Arrays.toCsvString(access.scope),","," ")`

Reviewer 2: do we use scope with a capital S in other places in the documentation? I feel we should use scope instead of Scope unless you made it this way for a particular reason.

Author: As we discussed, we might need to chat with Susan about capitalization of "scope", but I've remove the capital 'S' for now :)

## Expressions for OAuth2/OIDC custom claims

Okta provides a few expressions that you can only use with OAuth2/OIDC custom claims.
* See [Create claims](/docs/guides/customize-authz-server/create-claims).
* See [Include app-specific information in a custom claim](/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute).

| Syntax           | Definitions                                                             | Examples     |
| --------         | ----------                                                              | ------------ |
| `app.$attribute` | `app` refers to the name of the OIDC app.<br>`$attribute` refers to the attribute variable name. | app.id<br>app.clientId<br>app.profile |

> | `access.scope` | `access` refers to the access token requesting the scopes.<br>`scope` refers to the array of granted scopes. | access.scope |

Reviewer 3: ```suggestion
| `access.scope` | `access` refers to the access token that requests the scopes.<br>`scope` refers to the array of granted scopes. | access.scope |
```
